modules/blockexplorer/blockexplorer.go

```diff
@@ -0,0 +1,55 @@
+package blockexplorer
+
+import (
+	"errors"
+
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/sync"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// The blockexplorer module provides a glimpse into what the blockchain
+// currently looks like.
+
+// Basic structure to store the blockchain. Metadata may also be
+// stored here in the future
+type ExplorerState struct {
```

Comment:
Don't use the word 'State' in your structs. I think 'BlockExplorer' is a better name.

The consensus package has a 'State', but that's only because we hadn't made the convention yet. I'm in the process of renaming it to 'ConsensusSet', and that change is going to be completed once I've finished thoroughly testing the package.